language/stdlib/modules/libra_configs.move

```diff
@@ -0,0 +1,40 @@
+address 0x0:
+module LibraConfig {
+    use 0x0::Transaction;
+    use 0x0::LibraSystem;
+
+    // A generic singleton resource that holds a value of a specific type.
+    resource struct T<U: copyable> { payload: U }
+
+    resource struct ReconfigurationCapabilityHolder {
+        capability: LibraSystem::ReconfigurationCapability,
+    }
+
+    public fun initialize() {
+        let sender = Transaction::sender();
+
+        // Only callable by the Association address for now.
+        Transaction::assert(sender == 0xA550C18, 1);
+        move_to_sender<ReconfigurationCapabilityHolder>(ReconfigurationCapabilityHolder {
+            capability: LibraSystem::grant_reconfiguration_capability(),
+        })
+    }
+
+    // This can only be invoked by the Association address, and only a single time.
+    public fun set<U: copyable>(payload: U) acquires T, ReconfigurationCapabilityHolder {
```

Comment:
Can we get rid of `ReconfigurationCapabilityHolder` by having this take a `&ReconfigurationCapability`?

Reply:
The main reason is that I was imagining that we might want to have a separate account for holding those flags so they might not live under `0xa550c18`. That's the whole reason why I was thinking about having a `ReconfigurationCapabilityHolder` so that the association will grant the capability once and the other address can use such capability to emit events in the future.

Reply:
Ah, I thought we had decided to keep everything under one address for now to keep things as simple as possible. For me, it's easier to solidify what makes sense from a permissions/simplicity perspective first, then think about space optimization once the important semantics are settled. 

Otherwise, it becomes hard for me to remember whether we put something under a particular address because it's crucial for correctness or for space reasons...